inference-engine/tests/functional/plugin/gna/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -64,5 +64,13 @@ std::vector<std::string> disabledTestPatterns() {
         R"(.*CachingSupport.*_batch2_.*)",
         // TODO: Issue 51525
         R"(.*CachingSupport.*KSOFunction.*)",
+        // TODO: Issue XXX (Param -> Result subgraphs)
```

Comment:
What about replacing `XXX` with the issue number? :)

Reply:
fixed